# HG changeset patch
# User Julien Bect <jbect@users.sourceforge.net>
# Date 1510502683 -3600
#      Sun Nov 12 17:04:43 2017 +0100
# Node ID 3ed869a14b560fb32183f25a267bead5739262b3
# Parent  a47e758f233dc130af6c1694661207497fb82e98
configure.ac: Add missing --system-libs arguments to llvm-config call

diff -r a47e758f233d -r 3ed869a14b56 configure.ac
--- a/configure.ac	Wed Oct 18 15:18:28 2017 +0200
+++ b/configure.ac	Sun Nov 12 17:04:43 2017 +0100
@@ -1652,7 +1652,8 @@
 
     LDFLAGS="$LDFLAGS $LLVM_LDFLAGS"
     LLVM_SO=LLVM-`$LLVM_CONFIG --version`
-    AC_CHECK_LIB([$LLVM_SO], [LLVMBuildAdd], [LLVM_LIBS="-l$LLVM_SO"], [LLVM_LIBS=`$LLVM_CONFIG --libs`])
+    AC_CHECK_LIB([$LLVM_SO], [LLVMBuildAdd], [LLVM_LIBS="-l$LLVM_SO"],
+                 [LLVM_LIBS="`$LLVM_CONFIG --libs` `$LLVM_CONFIG --system-libs`"])
 
     dnl
     dnl Define some extra flags that LLVM requires in order to include headers.
